src/tokio_netstring.rs

```diff
@@ -0,0 +1,90 @@
+#![allow(unused_imports)]
+#![allow(unused_variables)]
```

Comment:
I'll pretend I didn't see these...

Reply:
yeah those are fixed, just leftovers, it annoys the fuck out of ide,